src/main/java/net/imglib2/algorithm/convolution/fast_gauss/FastGauss.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * To the extent possible under law, the ImageJ developers have waived
+ * all copyright and related or neighboring rights to this code.
+ *
+ * See the CC0 1.0 Universal license for details:
+ *     http://creativecommons.org/publicdomain/zero/1.0/
+ */
+package net.imglib2.algorithm.convolution.fast_gauss;
+
+import net.imglib2.RandomAccessible;
+import net.imglib2.algorithm.convolution.Convolution;
+import net.imglib2.algorithm.convolution.LineConvolution;
+import net.imglib2.algorithm.convolution.MultiDimensionConvolution;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.RandomAccessibleInterval;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Faster alternative to {@link net.imglib2.algorithm.gauss3.Gauss3}.
+ * It's especially faster if sigma is larger than 3.
+ * <p>
+ * The result is a good approximation to "real" gauss. But the
+ * same holds for {@link net.imglib2.algorithm.gauss3.Gauss3}.
+ * Which one is better is not known so far.
```

Comment:
This comment is not very helpful IMO. Would be better to state what the difference in approximation is.
I.e. `Gauss3` does a (non-approximated) convolution with a Gaussian kernel truncated at 3*sigma+1.
What is the approximation here?

Also, you should add a reference to the paper that is implemented here.